hazelcast/src/main/java/com/hazelcast/map/impl/eviction/ExpirationManager.java

```diff
@@ -294,15 +295,23 @@ private boolean notHaveAnyExpirableRecord(PartitionContainer partitionContainer)
     }
 
     private Operation createExpirationOperation(int expirationPercentage, int partitionId) {
-        return new ClearExpiredOperation(expirationPercentage)
+        final ClearExpiredOperation clearExpiredOperation = new ClearExpiredOperation(expirationPercentage);
+        return clearExpiredOperation
                 .setNodeEngine(nodeEngine)
                 .setCallerUuid(nodeEngine.getLocalMember().getUuid())
                 .setPartitionId(partitionId)
                 .setValidateTarget(false)
-                .setServiceName(SERVICE_NAME);
+                .setServiceName(SERVICE_NAME)
+                .setOperationResponseHandler(new OperationResponseHandler() {
+                    @Override
+                    public void sendResponse(Operation op, Object response) {
+                        assert response instanceof CallTimeoutResponse;
```

Comment:
Is a `CallTimeoutResponse` the only kind of response expected here?

Reply:
yes it is the only kind of expected exception.